packages/jest-circus/runner.js

```diff
@@ -8,3 +8,8 @@
 // Allow people to use `jest-circus/runner` as a runner.
 const runner = require('./build/legacy-code-todo-rewrite/jestAdapter');
 module.exports = runner;
+
+exports = require('./build/state');
```

Comment:
does this actually work? I think you need to do `module.exports.state = require('./build/state');` etc